protos/sync.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+
+import "blockchain.proto";
+
+message SyncRequest {
+    repeated uint64 heights = 1;
+}
+
+message SyncResponse {
+    repeated Block blocks = 1;
+}
```

Comment:
feed new line
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file